gitlab/v4/objects/merge_request_approvals.py

```diff
@@ -89,6 +89,8 @@ def set_approvers(
         approver_ids: Optional[List[int]] = None,
         approver_group_ids: Optional[List[int]] = None,
         approval_rule_name: str = "name",
+        *,
+        approver_usernames: List[str] = [],
```

Comment:
No, this is not correct Python usage. There are articles [1] out there why not use `[]` as a default value.

Do, like was done in the very first revision:

```
approver_usernames: Optional[List[str]] = None
```

[1] https://nikos7am.com/posts/mutable-default-arguments/

Reply:
Done